ci/do_ci.sh

```diff
@@ -505,7 +505,7 @@ elif [[ "$CI_TARGET" == "deps" ]]; then
 
   exit 0
 elif [[ "$CI_TARGET" == "verify_examples" ]]; then
-  run_ci_verify "*" "wasm-cc|win32-front-proxy|shared"
+  run_ci_verify "*" "win32-front-proxy|shared"
```

Comment:
I may be missing some context, why is `wasm-cc` not being validated anymore?
it's still part of the examples, so should be validated, no?

Reply:
in this line its not being _excluded_

it is being validated now in this test - just not separately